OptimizelySDK/Event/Builder/EventBuilder.cs

```diff
@@ -35,7 +35,9 @@ public class EventBuilder
 
         private const string ACTIVATE_EVENT_KEY = "campaign_activated";
 
-        public const string RESERVED_ATTRIBUTE_KEY_BUCKETING_ID_EVENT_PARAM_KEY = "optimizely_bucketing_id";
+        public const string BOT_FILTERING_ATTRIBUTE = "$opt_bot_filtering";
+
+        public const string USER_AGENT_ATTRIBUTE = "$opt_user_agent";
```

Comment:
nit. Like in other SDKs, lets consolidate all of the "special" attributes i.e. bucketing ID, bot filtering and user agent attributes.